src/layer.ts

```diff
@@ -94,9 +94,16 @@ export function findHandlers(service: Service, exclude: string[], defaultRuntime
     ) as FunctionInfo[];
 }
 
-export function applyLambdaLibraryLayers(service: Service, handlers: FunctionInfo[], layers: LayerJSON) {
+export function applyLambdaLibraryLayers(
+  service: Service,
+  handlers: FunctionInfo[],
+  layers: LayerJSON,
+  accountId?: string,
+) {
   const { region } = service.provider;
-  const regionRuntimes = layers.regions[region];
+  // It's possible a local account layer is being used in a region we have not published to so we use a default region's ARNs
+  const shouldUseDefaultRegion = layers.regions[region] === undefined && accountId !== undefined;
```

Comment:
Just making sure I understand, we are using a fallback region `DEFAULT_REGION` to simply get a list of layers to make sure we can buid them with the users account id, assumming they have deployed them in the region they set in `provider.region|function.region`. Is this correct?

Reply:
yup, that's exactly right 🥇